develop/DSI/DSIOpenConfig/DSI_OpenConfig_Middleware.md

```diff
@@ -405,6 +419,14 @@ public static object ConvertEpochTimeUtcTicksToOleAutomationTime(DataMinerConnec
 }
 ```
 
+#### Complex value
```

Comment:
@LaurensMT, these two titles (line 422 & 426) don't really seem to fit here — or at least not at this header level. All the other level-4 headers here are how-to instructions, while these two seem to be reference sections about specific values. Should these actually be level-5 headers meant as subsections of "Converting the value before doing the set"? Or otherwise, could you adjust them so they are also how-to instructions, in line with the other level-4 headers here?

Reply:
@MariekeGO, I've changed it into level-5 headers